NewAndNoteworthy/CHANGELOG-API.md

```diff
@@ -24,6 +24,11 @@ The following classes have been removed or modified in API breaking ways:
   - spelling corrected for methods with Uninitialized in the name
   - setWarnUnused renamed to setWarnUnusedVars and isWarnUnused renamed to isWarnUnusedVars
 
+### StandardBuildConfiguration.setBuildCommand(String[]) and StandardBuildConfiguration.setCleanCommand(String[]) removed
```

Comment:
@ewaterlander I have gone back and forth a little on straight out removing this API in favour of my new code. Please let me know if you are depending on the array version of setBuildCommand and setCleanCommand.

Reply:
Hi Jonah,

I'm not depending on the array versions of these methods.